.gitignore

```diff
@@ -15,10 +15,10 @@
 /.eggs
 /.tox
 /__pycache__
-/build/
+**/build/
```

Comment:
I suggest if there are changes needed to the .gitignore to keep them in a separate PR.

Reply:
Moved .gitignore changes to https://github.com/google/timesketch/pull/2311